site/oauth2-examples-azure.md

```diff
@@ -118,6 +118,13 @@ Now that some roles have been created for your application, you still need to as
 
 9. Repeat the operations for all the roles you want to assign.
 
+## About custom signing keys
```

Comment:
How about changing the heading Optional: Create Custom Signing Keys 
(remember all caps on the 1st letters of main words in headings)